tests/dummy/app/routes/components/fixed-top-nav-fixed-width-body-tabs/template.hbs

```diff
@@ -0,0 +1,41 @@
+<h2>Fixed Top Nav w/ Fixed Width Body Tabs</h2>
+
+{{!- BEGIN-SNIPPET fixed-top-nav-fixed-width-body-tabs }}
+<page class="example-layout page1">
+  <hbox fit justify="between" class="blue header">
```

Comment:
We had a discussion in slack about this. You could land either way. The container adds an element, but is more explicit. The justify between seems more flexboxy and is less code, but not easier to understand. Both ways work.